misc/json-to-record-converter/src/main/java/io/ballerina/jsonmapper/JsonToRecordMapper.java

```diff
@@ -117,7 +116,7 @@ public static JsonToRecordResponse convert(String jsonString, String recordName,
      * @return {@link JsonToRecordResponse} Ballerina code block or the Diagnostics
      */
     public static JsonToRecordResponse convert(String jsonString, String recordName, boolean isRecordTypeDesc,
-                                               boolean isClosed, boolean forceFormatRecordFields, Path filePath) {
+                                               boolean isClosed, boolean forceFormatRecordFields, String filePath) {
```

Comment:
Any specific reason to change the parameter type from `Path` to `String` in here?

Reply:
Underlying serializer of LSP(gson) cannot serialize String/URI to `java.nio.file.Path` since it's an interface. And even now I realized to use URI instead of String as no intermediate conversion required then (we discussed that the request will contain filePath URI)